src/components/skills-quiz/TagCloud.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Card } from '@edx/paragon';
+
+const TagCloud = ({ tags, onRemove }) => (
+  <>
+    <Card style={{ width: '45%' }}>
+      <Card.Body>
+        <Card.Text>
+          <ul className="item">
+            {
+              tags.map(
+                tag => (
+                  <li className="list-item">
```

Comment:
Can we please add key attribute here so that react can uniquely identify each element. `<li className="list-item" key={tag.title}>`